client/src/pages/Login.tsx

```diff
@@ -0,0 +1,156 @@
+import * as React from 'react';
+import Joi from 'joi';
+
+import axios from 'axios';
+import {
+  Avatar,
+  Button,
+  CssBaseline,
+  TextField,
+  Link,
+  Grid,
+  Box,
+  Typography,
+  Container,
+  FormControlLabel,
+  Checkbox,
+} from '@mui/material';
+import LockOutlinedIcon from '@mui/icons-material/LockOutlined';
+
+const Copyright = (props: any) => {
```

Comment:
Isn't this the same as the Copyright component on the signup page, remember to add it once and call it here also.

Reply:
done yaa brooo @moustf